functions/sheets.js

```diff
@@ -14,33 +15,35 @@ class Header {
     this.name = suffix[1]
   }
 
-  parse(val) {
+  parse(value) {
+    let val = value || ''
     if (this.type == 'number') {
       return +val
     }
     if (this.type == 'list') {
-      return val.split(', ')
+      return val.split(',').map(s => s.trim())
     }
-    if (val === null) {
-      return ''
-    }
-    return val
+    return val.trim()
   }
 
   get(row) {
-    return this.parse(row[this.value])
+    return this.parse(row.get(this.value))
```

Comment:
This I could not explain, but `row[headerValue]` would return undefined; and the documentation [here](https://theoephraim.github.io/node-google-spreadsheet/#/?id=working-with-rows) mentions `.get`, which does work in this case.
Not sure why it worked before, but do you see any reason to stick to `[]`?

Reply:
No idea. If this works then great :)